packages/docs/source/components/status.md

```diff
@@ -0,0 +1,211 @@
+# Status
+
+Status is used to keep users informed, by providing appropriate feedback on particular concepts or entities. For Okta, status can be used to display overall operational status all the way down to very granular concepts, like user status.
+
+## Usage
+
+Status should be used to communicate the state of a discrete item, such as a server or individual process. Both labeled and unlabeled variants are acceptable, but should follow the associated guidelines below.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <dl class="ods-status is-ods-status-success">
+      <dt class="ods-status--label">
+        Server status
+      </dt>
+      <dd class="ods-status--value">
+        System operational
+      </dd>
+    </dl>
+  </div>
+
+  ```html
+  <dl class="ods-status is-ods-status-success">
+    <dt class="ods-status--label">
+      Server status
+    </dt>
+    <dd class="ods-status--value">
+      System operational
+    </dd>
+  </dl>
+  ```
+</figure>
+
+Labels may be hidden by applying the `.is-ods-status-label-hidden` class. If a label is not visually present, ensure that appropriate context is communicated by the Status itself and close proximity to the content that it’s supporting.
```

Comment:
@arnoldsandoval-okta any thoughts on this class name? This is precise, but a little verbose.

Reply:
is this any different than `u-visually-hidden`? I would prefer to stick to that as much as possible if its not.

Reply:
This appears to be the same as `u-visually-hidden`, I would prefer we leverage that wherever possible

Reply:
Currently, the declared props are identical, but I try to keep component classes/states/variants and our utilities separate in case we want to use a different implementation in the future - styling can change without markup and binding layers being updated.

Reply:
That makes sense. The class name is way too long. Still, I'm inclined to say keep it as is for now. We should surface this during the naming/conventions meeting to see if there's a different approach we want to take in this instance.